build_tools/cmake/iree_lit_test.cmake

```diff
@@ -82,9 +82,10 @@ function(iree_lit_test)
   )
 
   list(APPEND _RULE_LABELS "${_PACKAGE_PATH}")
-  set_property(TEST ${_NAME_PATH} PROPERTY ENVIRONMENT "TEST_TMPDIR=${_NAME}_test_tmpdir")
   set_property(TEST ${_NAME_PATH} PROPERTY LABELS "${_RULE_LABELS}")
   set_property(TEST ${_NAME_PATH} PROPERTY REQUIRED_FILES "${_TEST_FILE_PATH}")
+  set_property(TEST ${_NAME_PATH} PROPERTY ENVIRONMENT "TEST_TMPDIR=${_NAME}_test_tmpdir")
+  iree_add_test_environment_properties(${_NAME_PATH})
```

Comment:
Should we move the TEST_TMPDIR stuff into this function?

Reply:
I started that way, but decided against it (at least in this change) seeing that it's handled slightly differently between test functions:

https://github.com/google/iree/blob/4fe20045d7b72f82dba7bb9d4bad31cadbf22673/build_tools/cmake/iree_check_test.cmake#L135

https://github.com/google/iree/blob/4fe20045d7b72f82dba7bb9d4bad31cadbf22673/build_tools/cmake/iree_cc_test.cmake#L133

https://github.com/google/iree/blob/4fe20045d7b72f82dba7bb9d4bad31cadbf22673/build_tools/cmake/iree_cc_test.cmake#L148

https://github.com/google/iree/blob/4fe20045d7b72f82dba7bb9d4bad31cadbf22673/build_tools/cmake/iree_lit_test.cmake#L85

Reply:
Huh we should probably clean that up, but let's punt for now